lib/OpenQA/Worker/Job.pm

```diff
@@ -913,8 +916,8 @@ sub post_upload_progress_to_liveviewhandler {
 
     my $job_id = $self->id;
     $self->client->send(
-        post => "/liveviewhandler/api/v1/jobs/$job_id/upload_progress",
-        service_port_delta => 2,                     # liveviewhandler is supposed to run on web UI port + 2
+        post               => "/liveviewhandler/api/v1/jobs/$job_id/upload_progress",
+        service_port_delta => 2,                                                        # liveviewhandler is supposed to run on web UI port + 2
```

Comment:
the additional indention on the comment looks broken, please try to put it above the line

Reply:
I just reverted everything. this was mistake to include unrelated lines formatting   into this PR